### DISCUSSION TITLE
Help with White paper translation

### DISCUSSION THREAD

**Discussion by memoryzera** [teaxyz/white-paper#119] [Q&A]
Hey guys! I'm trying to translate the White Paper to pt-br, but I'm getting the following error when trying to work on my fork:

gitpod /workspace/white-paper (main) $ git checkout -b i18n/pt-br v1.0.5
fatal: 'v1.0.5' is not a commit and a branch 'i18n/pt-br' cannot be created from it

Can you guys help me fix that?

Edit: Fixed it! :)

**Comment by renatghoste** [comment]
how u fixed that?

**Reply by thomas-borrel** [reply]
Hi. Just checking that you are translating the latest version which is in Gitbook
